condition/scenario_conditions/include/scenario_conditions/condition_visualizer.h

```diff
@@ -11,14 +12,15 @@ class ConditionManager;
 class ConditionVisualizer
 {
 public:
-  ConditionVisualizer();
+  ConditionVisualizer(const rclcpp::Node::SharedPtr node);
   void publishMarker(ConditionManager& manager);
 
 private:
   void addMarker(std::string name, bool result, bool is_success_condition);
 
-  ros::Publisher pub_marker_;
-  visualization_msgs::MarkerArray marker_array_;
+  rclcpp::Node::SharedPtr node_;
```

Comment:
[nit] you could probably make this `const`, or do we want to change the node ever?

Reply:
None of the other members in this package are const even though they could be. I think it's better to stay consistent for now.

Reply:
ok, let's just keep in mind for autoware.auto merges to look at const correctness

Reply:
Agree